dd-sdk-android/src/main/kotlin/com/datadog/android/rum/internal/domain/scope/RumViewManagerScope.kt

```diff
@@ -243,12 +216,6 @@ internal class RumViewManagerScope(
             RumRawEvent.StartAction::class.java,
             RumRawEvent.StartResource::class.java
         )
-        internal val validAppLaunchEventTypes = arrayOf<Class<*>>(
```

Comment:
we need also to delete `Forge.validAppLaunchEvent()` in `RumRawEventExt` file in tests